### PR TITLE
[FranceInter] Extractor Update

### PR DESCRIPTION
**Pull request by renalid**
Update of the "FranceInter" radio extractor : webpages HTML structure had changed, the extractor didn't work. So I updated this extractor to get the mp3 URL and all details.

here is an example : https://www.franceinter.fr/emissions/la-tete-au-carre/la-tete-au-carre-30-juin-2016

[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.07.02
[debug] Git HEAD: 7191f5e
[debug] Python version 2.7.12 - Darwin-15.4.0-x86_64-i386-64bit
[debug] exe versions: none
[debug] Proxy map: {}

Thanks !

### CODE REVIEW

File: youtube_dl/extractor/franceinter.py [hunk 1]
```diff
@@ -2,21 +2,27 @@
...
+            'id': 'la-tete-au-carre/la-tete-au-carre-30-juin-2016',
             'ext': 'mp3',
-            'title': 'L’Histoire dans les jeux vidéo',
-            'description': 'md5:7e93ddb4451e7530022792240a3049c7',
-            'timestamp': 1387369800,
-            'upload_date': '20131218',
+            'title': 'Regards sur le sport du 30 juin 2016 - France Inter',
+            'description': 'UEFA Europa, Jeux Olympiques... La période est aux sports, dans les gradins ou devant les écrans. Mais quel est le regard des spécialistes sur cette pratique? ',
+            'timestamp': 1467244800,
+            'upload_date': '20160630',
```
Comment (dstftw) [inline youtube_dl/extractor/franceinter.py]: Keep the old test.

File: youtube_dl/extractor/franceinter.py [hunk 2]
```diff
@@ -25,17 +31,18 @@ def _real_extract(self, url):
...
-        description = self._html_search_regex(
-            r'<span class="description">(.*?)</span>',
-            webpage, 'description', fatal=False)
-        timestamp = int_or_none(self._search_regex(
-            r'data-date="(\d+)"', webpage, 'upload date', fatal=False))
+        title = self._og_search_title(webpage)
+        description = self._og_search_description(webpage)
+        
+        extractdate = self._search_regex(
+            r'([0-9]+[.][0-9]+[.][0-9]+)', video_url, 'extractdate', fatal=False)
```
Comment (dstftw) [inline youtube_dl/extractor/franceinter.py]: This captures just the first date found.

**Comment by dstftw** [comment]
Check code with flake8 and squash commits.

**Comment by renalid** [comment]
Thanks @dstftw for your review.
I changed the test part to get the same original content and I changed how to get the right upload date by converting french month name.

**Comment by renalid** [comment]
Hello @dstftw 
As you adviced, I checked code with flake8 and I commited changes.
Thanks

**Comment by renalid** [comment]
Hello,

What should I do in order that this pull request is validated ?

Thanks

**Comment by dstftw** [comment]
Superseded by #10538.

**Pull request closed without merging**